config/static_test.go

```diff
@@ -30,6 +30,7 @@ BlackListed:
     myIP.ms: true
     MalwareDomains.com: true
     MalwareDomainList.com: true
+		BlacklistDatabase: "rita-bl"
```

Comment:
I think this is the spot the test is complaining about.  Just a tab vs. space issue.

Reply:
fixed :)